src/index.html

```diff
@@ -7,8 +7,42 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+        src="images/logo.png"
+        alt="logo"
+        href="#"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a class="nav__link is-active" href="#apple">apple</a></li>
+          <li class="nav__item"><a class="nav__link" href="#samsung">samsung</a></li>
+          <li class="nav__item"><a class="nav__link" href="#tablets">smartphones</a></li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              ref="#laptopscomputers"
+              data-qa="hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link"
+            href="#gadgets">Gadgets</a>
```

Comment:
```suggestion
            <a class="nav__link" href="#gadgets">Gadgets</a>
```
place attr on new lines when >= 3 attr or line has > 80 symbols